integration_tests/test-requirements.txt

```diff
@@ -6,3 +6,4 @@ kombu
 openapi-spec-validator<0.6.0  # dependency conflict on requests version (>2.31.0) with wazo clients (=2.25.1)
 pyhamcrest
 pytest
+setuptools
```

Comment:
We don't have this explicit dependency in any other repo.
Any reason to justify to add it? (and why only in test-requirements?)

Reply:
https://github.com/wazo-platform/wazo-amid/pull/89